setup.py

```diff
@@ -29,26 +29,29 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 
-from setuptools import setup
-from pyseq import __version__
-
 from os import path
+from setuptools import setup, find_packages
 
 this_directory = path.abspath(path.dirname(__file__))
 with open(path.join(this_directory, "README.md")) as f:
     long_description = f.read()
 
 setup(
     name="pyseq",
-    version=__version__,
+    version="0.7.0",
```

Comment:
I think it's just a little simpler to use the `__version__` in case they get out of sync.

Reply:
yes I struggled with a good way to import `__version__` and gave up heh. I have seen a common way to do this is to add a version.py module.

Reply:
That's understandable.  I've seen people use the version.py files as well.  You could read the __version__ from the module file.  Similar to how you get the long description from the README

Reply:
pushed an update to get the version from the package, and bumped version to 0.8